### PR TITLE
Plans overhaul Phase 1

### PR DESCRIPTION
**Pull request by taggon**
#### Changes proposed in this Pull Request

* This is a part of the first phase of the Plan Overhaul project p58i-c1P-p2 .
* With this PR, a new plan `Managed` will be introduced.

#### Testing instructions

Before testing, please make sure:

* You are proxied.
* D75519-code is applied to your sandbox.
* `public-api.wordpress.com` is sandboxed.

Ready? Let's start.

* Signup process
  * Going to `/start/new` as a new user.
  * You should be able to see the new plans in the plans step.
  * Create_a_site_—_WordPress_com
  * Check it looks good on small screen as well.
  * Whatever you chose, the process should be completed without any errors.
* Admin `/plans` page when your site is on a free plan.
  * Plans_‹_Site_Title_—_WordPress_com
  * You should be able to purchase the managed plan with the `Select Managed` button.
* Admin `/plans` page when your site is on the new managed plan.  
  * Plans_‹_Site_Title_—_WordPress_com
  * Click `View plan` and you will see the purchase information.
  * Purchase_Settings_‹_Site_Title_—_WordPress_com
* Admin `/plans` page when your site is on a legacy plan such as Personal, Premium, Business, and eCommerce.
  * Nothing should change.

Related to #

### CODE REVIEW

**Review by claudiucelfilip — APPROVED** [review]
The PR mostly worked as described.
This should be enough to iterate upon. This is a list of things I noticed, but we can address them in a follow-up. 
- the `View Plan` button in the /plans page often leads to the `checkout` page instead of `payments`
- Managed plan column gets squished on narrow screen widths
- cross icon for `Not Included` features should be red
- the plan title and prices should have the same font as the page title
- the page title and subtitle changes based on screen width
- the table items should be vertically middle-aligned
- some tooltips don't have content or have placeholder text (already raised, WIP)

LGTM

File: client/my-sites/plans-comparison/plans-comparison-action.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+	if ( isCurrentPlan ) {
+		return props.canPurchase ? translate( 'Manage plan' ) : translate( 'View plan' );
+	}
+
+	return translate( 'Select %(plan)s', {
+		args: {
+			plan: plan?.getTitle(),
+		},
+		context: 'Button to select a paid plan by plan name, e.g., "Select Personal"',
```
Comment (southp) [inline client/my-sites/plans-comparison/plans-comparison-action.tsx]: This context should be a comment instead. I'll fix it shortly.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7163542

Thank you @taggon for including a screenshot in the description! This is really helpful for our translators.

**Comment by enejb** [comment]
I noticed that on the mobile web this looks a bit broken:

See



cc: @taggon and @southp

File: client/my-sites/plan-price/index.jsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { getCurrencyObject } from '@automattic/format-currency';
 import classNames from 'classnames';
 import { localize } from 'i18n-calypso';
+import { isNumber } from 'lodash';
```
Comment (tyxla) [inline client/my-sites/plan-price/index.jsx:4]: Is there any chance we could remove the usage of Lodash `isNumber`? This is currently the only use of this method and we got rid of all of them a while ago in #52156. Let me know if you need any help 🙌

**Comment by claudiucelfilip** [comment]
@enejb @tyxla Thanks for bringing up these issues. 
I included fixes for them in the follow-up PR to implement the design for the Plans overhaul: #61757

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7163542).

**Pull request merged**